conf/solr/conf/managed-schema.xml

```diff
@@ -557,7 +557,7 @@
     </fieldType>
 
     <!-- A text field that applies standard title sorting conventions to its content -->
-    <fieldType name="text_title_sort" class="solr.TextField" sortMissingLast="false" omitNorms="true">
+    <fieldType name="text_title_sort" class="solr.SortableTextField" sortMissingLast="false" omitNorms="true">
```

Comment:
SortableTextFields get sorted on the stored value, which is pre-analysis string, so doesn't include the article processing, case folding, or diacritic folding which the analysis pipeline does.